src/logic/operate.js

```diff
@@ -0,0 +1,25 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const bigOne = Big(numberOne);
+  const bigTwo = Big(numberTwo);
+  let result;
+  if (operation === '+') {
+    result = bigOne.plus(bigTwo).toString();
+  }
+  if (operation === '-') {
+    result = bigOne.minus(bigTwo).toString();
+  }
+  if (operation === 'X') {
+    result = bigOne.times(bigTwo).toString();
+  }
+  if (operation === '÷') {
+    result = bigOne.div(bigTwo).toString();
+  }
```

Comment:
dont forget to handle DIVISION BY ZERO in next milestone, else it will generate an error

Reply:
alright! thanks for the information